pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -209,6 +213,14 @@ RUN pip install --no-cache-dir -U \
     "plotly>=5.1,<6" \
     "seaborn>=0.11,<1" \
     "shap>=0.39,<1"
+    # Install PyHive, Sparkmagic and Astra libraries
+    "boto3>1.0<2.0" \
+    "sasl<1" \
+    "thrift_sasl<1" \
+    "PyHive<1"
+    "sparkmagic<1" \
```

Comment:
right now safety check doesnt run on PR, comment out this code to run it on this PR https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_safety_check.py#L151:L158